SDWebImage/Core/SDImageIOCoder.m

```diff
@@ -95,22 +96,22 @@ + (UIImage *)createBitmapPDFWithData:(nonnull NSData *)data pageNumber:(NSUInteg
     CGAffineTransform scaleTransform = CGAffineTransformMakeScale(xScale, yScale);
     CGAffineTransform transform = CGPDFPageGetDrawingTransform(page, box, drawRect, 0, preserveAspectRatio);
     
-    SDGraphicsBeginImageContextWithOptions(targetRect.size, NO, 0);
```

Comment:
This change does not do anything performance benefit. `UIGraphicsImageRenderer` should be only used **When render this CGImage into screen, need dynamic bitmap info**

The encode or export usage, just create CGBitmapContext or CGPDFContext for your own

Reply:
And, read the comment I wrote here:

https://github.com/SDWebImage/SDWebImage/blob/5.15.3/SDWebImage/Core/SDGraphicsImageRenderer.h#L13